packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -73,7 +72,7 @@ class TeamsFxTokenCredential implements TokenCredential {
       if (this.tokenCredentialBase) {
         const token = await this.tokenCredentialBase.getToken();
         const tokenJson = ConvertTokenToJson(token.accessToken);
-        if (scopes === Constants.azureSqlScope) {
```

Comment:
Is this PR removes the whole SQL feature? If so, do we still have the scenario to get sql token? Or the sql token part can be removed as well?

Reply:
Just let us know so we can remove this on login code clean.

Reply:
Thanks. 
Since the whole SQLfeature should be removed, I will remove this condition as well.